jsonview/test_jsonview.py

```diff
@@ -308,6 +298,24 @@ def temp(req):
         eq_(200, res.status_code)
         eq_(payload, res.content)
 
+    @override_settings(
+        JSON_OPTIONS={'cls': 'jsonview.test_jsonview.CustomTestEncoder'})
+    def test_json_custom_serializer_string__no_for_json_function(self):
```

Comment:
I added this test so that coverage was still 100%, I don't know if I need the test or to remove the code since this is a compatibility thing.